BUSRA_ERDOGAN.md

```diff
@@ -0,0 +1,9 @@
+Hi, My name is Busra
+
+My favorite sport is swimming. I love swimming because when I swam always feel free.
+
+My favorite food is stuffed leaves from Turkish cuisine. Actually I love all Turkish dishes.
+
+If you can click the link, you can find my favorite music. https://www.youtube.com/watch?v=IugGpMrUrjE
```

Comment:
Try to use [markdown links](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links), not just plain URLs